app/controllers/settings/profiles_controller.rb

```diff
@@ -14,7 +14,7 @@ class Settings::ProfilesController < ApplicationController
   def show; end
 
   def update
-    if @account.update(account_params)
+    if UpdateAccountService.new.call(@account, account_params)
```

Comment:
I fixed, but I think using `raise_error` option to detect `:update` or `update!` is not good.
Do you have any idea?